brave/browser-laptop
# Adds promo

## Submitter Checklist:


## Test Plan:
- create folder `brave-development` and add `promoCode` file
- start browser and make sure that new tab with promo page is opened
- visit partner page and make sure that partner header is in the request via dev tools

## Reviewer Checklist:


Tests
